docs/ingest/import/spatial.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Geospatial
+---
+
+## Overview
+
+This code block demonstrates how to query a geospatial file.
+
+```sql
+CREATE FOREIGN DATA WRAPPER <wrapper_name>
+HANDLER spatial_fdw_handler
+VALIDATOR spatial_fdw_validator;
+
+CREATE SERVER <server_name>
+FOREIGN DATA WRAPPER <wrapper_name>;
+
+CREATE FOREIGN TABLE <table_name> ()
+SERVER <server_name>
+OPTIONS (files '<files>');
+```
+
+<Accordion title = "Example Usage">
+```sql
+CREATE FOREIGN DATA WRAPPER spatial_wrapper
+HANDLER spatial_fdw_handler
+VALIDATOR spatial_fdw_validator;
+
+CREATE SERVER spatial_server
+FOREIGN DATA WRAPPER spatial_wrapper;
+
+CREATE FOREIGN TABLE spatial_table ()
+SERVER spatial_server
+OPTIONS (files 's3://bucket/folder/file.geojson');
+
+````
+</Accordion>
+
+<ParamField body="wrapper_name" required>
+  Foreign data wrapper name. Can be any string.
+</ParamField>
+<ParamField body="server_name" required>
+  Foreign server name. Can be any string.
+</ParamField>
+<ParamField body="table_name" required>
+  Foreign table name. Can be any string.
+</ParamField>
+<ParamField body="files" required>
+The path to the spatial file. For instance, `s3://bucket/folder/file.geojson` if the spatial file is in Amazon S3 or
+`/path/to/folder/file.geojson` if the spatial file is on the local file system.
+</ParamField>
+
+
+## Geospatial Options
+
+There are a number of options that can be passed into the `CREATE FOREIGN TABLE` statement.
+These are the same [options](https://duckdb.org/docs/extensions/spatial#st_read--read-spatial-data-from-files) accepted
+by DuckDB's `st_read` function in the `spatial` extension.
+
+```sql
+CREATE FOREIGN TABLE spatial_table ()
+SERVER spatial_server
+OPTIONS (
+    files 's3://bucket/folder/file.geojson',
+    layer 'layer_name'
+);
+````
```

Comment:
you have 4 backticks as closing in a few places, idk if that causes rendering issues